src/main/java/mil/dds/anet/beans/Report.java

```diff
@@ -329,6 +351,26 @@ public ReportPerson getPrimaryPrincipal() {
     return primaryPrincipal.getForeignObject();
   }
 
+  @GraphQLQuery(name = "authors")
+  public CompletableFuture<List<ReportPerson>> loadAuthors(
+      @GraphQLRootContext Map<String, Object> context) {
+    if (authors != null) {
+      return CompletableFuture.completedFuture(authors);
+    }
+    return loadReportPeople(context) // Force the load of reportPeople
+        .thenApply(l -> {
+          final List<ReportPerson> o =
+              l.stream().filter(p -> p.isAuthor()).collect(Collectors.toList());
```

Comment:
lambda can be replaced with method reference i.e filter(ReportPerson::isAuthor) as it is easier to read.